### PR TITLE
Update Redis Quickstarts

### PR DESCRIPTION
**Pull request by yunusemreozdemir**
This PR adds:
- Updated AWS Lambda quickstart with AWS CDK.
- Updated Next.js 14 quickstarts for app and pages routers.
- Updated Vercel Functions quickstarts for app and pages routers.
- FastAPI quickstart.
- Python AWS Lambda quickstart.
- Vercel Python Runtime quickstart using django.
- SST v2 quickstart.
- Ion quickstart.
- Azure Functions quickstart.

### CODE REVIEW

File: redis/quickstarts/nextjsapprouter.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```shell .env
+UPSTASH_REDIS_REST_URL=<YOUR_URL>
+UPSTASH_REDIS_REST_TOKEN=<YOUR_TOKEN>
+```
+
+### Home Page Setup
+
+Update `/app/page.tsx`:
+```tsx /app/page.tsx
+import { Redis } from "@upstash/redis";
```
Comment (ogzhanolguncu) [inline redis/quickstarts/nextjsapprouter.mdx:28]: Don't we have to specify its "use server" here?

Comment (yunusemreozdemir) [reply redis/quickstarts/nextjsapprouter.mdx:28]: I think Next.js uses Server Components by default in App Router so this should work fine. If you think there is a need for or a benefit in using Server Actions I can change.

File: mint.json [hunk 2]
```diff
@@ -263,8 +263,20 @@
           "group": "Quickstarts",
           "pages": [
             "redis/quickstarts/awslambda",
-            "redis/quickstarts/vercel",
-            "redis/quickstarts/nextjs13",
+            {
+              "group": "Vercel Functions",
+              "pages": [
+                "redis/quickstarts/vercelfunctionsapprouter",
```
Comment (fahreddinozcan) [inline mint.json:269]: we can split the path keywords, maybe like `vercel-functions-app-router`

File: mint.json [hunk 3]
```diff
@@ -273,7 +285,20 @@
...
+              "group": "SST",
+              "pages": [
+                "redis/quickstarts/sstv2"
+              ]
+            },
+            {
+              "group": "Python",
+              "pages": [
+                "redis/quickstarts/fastapi",
+                "redis/quickstarts/vercelpythonruntime"
```
Comment (fahreddinozcan) [inline mint.json:299]:
```suggestion
                "redis/quickstarts/vercel-python-runtime"
```

File: redis/quickstarts/nextjsapprouter.mdx (hunk 1 above)
Comment (ogzhanolguncu) [reply redis/quickstarts/nextjsapprouter.mdx]: Nah. I'm just making sure.

**Review by ogzhanolguncu — APPROVED** [review]

**Review by ytkimirti — APPROVED** [review]

**Pull request merged**